includes/wp-admin/class-welcome-fields.php

```diff
@@ -0,0 +1,274 @@
+		add_action( 'load-settings_page_activitypub', array( self::class, 'register_welcome_fields' ) );
+	}
+
+	/**
+	 * Register welcome fields.
+	 */
+	public static function register_welcome_fields() {
+		// Add settings sections.
+		add_settings_section(
+			'activitypub_intro',
+			\__( 'Welcome', 'activitypub' ),
+			array( self::class, 'render_welcome_intro_section' ),
+			'activitypub_welcome'
+		);
+
+		add_settings_section(
+			'activitypub_bookmarklet',
+			\__( 'Bookmarklet', 'activitypub' ),
+			array( self::class, 'render_bookmarklet_section' ),
+			'activitypub_welcome'
+		);
+
+		if ( ! is_user_disabled( Actors::BLOG_USER_ID ) ) :
+			add_settings_section(
+				'activitypub_blog_profile',
+				\__( 'Blog profile', 'activitypub' ),
+				array( self::class, 'render_blog_profile_section' ),
+				'activitypub_welcome'
+			);
+		endif;
+
+		if ( ! is_user_disabled( get_current_user_id() ) ) :
+			add_settings_section(
+				'activitypub_author_profile',
+				\__( 'Author profile', 'activitypub' ),
+				array( self::class, 'render_author_profile_section' ),
+				'activitypub_welcome'
+			);
+		endif;
+
+		add_settings_section(
+			'activitypub_troubleshooting',
+			\__( 'Troubleshooting', 'activitypub' ),
+			array( self::class, 'render_troubleshooting_section' ),
+			'activitypub_welcome'
+		);
+
+		if ( ACTIVITYPUB_SHOW_PLUGIN_RECOMMENDATIONS ) :
+			add_settings_section(
+				'activitypub_recommended_plugins',
+				\__( 'Recommended Plugins', 'activitypub' ),
+				array( self::class, 'render_recommended_plugins_section' ),
+				'activitypub_welcome'
+			);
+		endif;
+	}
+
+	/**
+	 * Render welcome intro section.
+	 */
+	public static function render_welcome_intro_section() {
+		?>
+		<p><?php echo wp_kses( \__( 'Enter the fediverse with <strong>ActivityPub</strong>, broadcasting your blog to a wider audience. Attract followers, deliver updates, and receive comments from a diverse user base on <strong>Mastodon</strong>, <strong>Friendica</strong>, <strong>Pleroma</strong>, <strong>Pixelfed</strong>, and all <strong>ActivityPub</strong>-compliant platforms.', 'activitypub' ), array( 'strong' => array() ) ); ?></p>
+		<?php
+	}
+
+	/**
+	 * Render bookmarklet section.
+	 */
+	public static function render_bookmarklet_section() {
+		?>
+		<p>
+			<?php
+			$bookmarklet_js = get_reply_intent_js();
+
+			/* translators: %s is the domain of this site */
+			$reply_from_template = __( 'Reply from %s', 'activitypub' );
+			$button              = sprintf(
+				'<a href="%s" class="button">%s</a>',
+				esc_attr( $bookmarklet_js ), // Need to escape quotes for the bookmarklet.
+				sprintf( $reply_from_template, \wp_parse_url( \home_url(), PHP_URL_HOST ) )
+			);
+			/* translators: %s is where the button HTML will be rendered. */
+			$button_and_explanation_template = \__(
+				'%s Save this bookmarklet to reply to posts on other sites from your own blog! When visiting a post on another site, click the bookmarklet to start a reply.',
```

Comment:
This could be an opportunity to fix #1433 and display the button above this string? There it wouldn't need to be accounted for for RTL languages either.